/joystick:
  ros__parameters:
    axis_max_val: 32767.0
    axis_deadzone_val: 300
    invert_y: true
    joy_topic_name: /joystick
    joy_path : /dev/input/js0